### ISSUE TITLE
Режим swing у кондиционера

### ISSUE THREAD

**Issue by Mrboob58** [AlexxIT/YandexStation#509]
Реально ли прокинуть режим swing (направление шторок) у кондиционера?

**Comment by AlexxIT** [comment]
Да. Но нужно вручную указать все нужные атрибуты устройства в yaml, согласно документации

**Comment by Mrboob58** [comment]
> Да. Но нужно вручную указать все нужные атрибуты устройства в yaml, согласно документации
Отлично. Буду пробовать, спасибо!

**Comment by Mrboob58** [comment]
Извиняюсь, а эти атрибуты нужно вносить в конфиг HA configuration.yml ?

**Comment by AlexxIT** [comment]
Да
